include/deal.II/hp/fe_collection.h

```diff
@@ -68,11 +68,56 @@ namespace hp
     explicit FECollection (const FiniteElement<dim,spacedim> &fe);
 
     /**
+     * Constructor. This constructor creates a FECollection from two
+     * base finite elements.
```

Comment:
Can you omit the word "base"? It has a particular meaning in the context of `FESystem` but that does not apply here.